Library/Formula/apkpatch.rb

```diff
@@ -0,0 +1,24 @@
+class Apkpatch < Formula
+  desc "AndFix provides a patch-making tool called apkpatch."
```

Comment:
This should be a description of `apkpatch`. Something like `desc "Patch-making tool for Android apps"` might work.